src/main/java/frc/robot/swerve/DriveController.java

```diff
@@ -7,4 +7,7 @@ public interface DriveController {
     public double getStateVelocity();
     // get wheel distance
     public double getWheelDistance();
+        
+    // update the smart dashboard
+    public void updateSmartDashboard(String sdPrefix);
 }
```

Comment:
As it turns out, the implementation of this method in both NeoDriveController and FalconDrive Controller are identical. THe differences are in the getStateVelocity methods. We could include a default implementation here and remove it from those classes to be overridden if needed in any future derived class.

Reply:
I figured it was better to let the real Drive controller do the monitoring, for future expansion. Also "DriveController" is an interface, so not sure we can add code.